app/views/users/two_factor_authentication_setup/_mfa_selection_component.html.erb

```diff
@@ -0,0 +1,28 @@
+<div class="mfa-selection">
+  <%= check_box_tag(
+        'two_factor_options_form[selection][]',
+        option.type,
+        option.type == 'phone' && flash[:phone_error].present?,
+        disabled: option.disabled?,
+        class: 'usa-checkbox__input usa-checkbox__input--tile',
+        id: "two_factor_options_form_selection_#{option.type}",
+      ) %>
+  <%= label_tag(
+        "two_factor_options_form_selection_#{option.type}",
+        class: [
+          option.type === 'phone' && flash[:phone_error] && 'checkbox__invalid',
+          'usa-checkbox__label',
+          'usa-checkbox__label--illustrated',
+        ].select(&:present?).join(' '),
+      ) do %>
+            <%= image_tag(asset_url("mfa-options/#{option.type}.svg"), alt: "#{option.label} icon", class: 'usa-checkbox__image') %>
+            <div class="usa-checkbox__label--text"><%= option.label %>
+              <span class="usa-checkbox__label-description">
+                <%= option.info %>
+              </span>
+            </div>
+      <% end %>
+      <% if option.type === "phone" && flash[:phone_error] %>
+          <%= render AlertComponent.new(type: :error, id: 'phone_error', class: 'margin-bottom-1 margin-top-neg-1 checkbox__alert', message: flash[:phone_error]) %>
```

Comment:
From what I can tell, how we're using this alert is as an error message, so seems we can just use the default styling, rather than creating new variant of alert.

https://design.login.gov/components/form-fields/#form-validation-errors

```suggestion
          <div id="phone_error" class="usa-error-message margin-bottom-1 margin-top-neg-1" role="alert">
            <%= flash[:phone_error] %>
          </div>
```

Reply:
agreed. I removed the unnecessary sass as well